eos/statistics/log-likelihood.hh

```diff
@@ -255,12 +259,22 @@ namespace eos
             // todo document
             static LogLikelihoodBlockPtr Mixture(const std::vector<LogLikelihoodBlockPtr> & components,
                                                  const std::vector<double> & weights);
+
             /*!
              * Create a new LogLikelihoodBlock for n observables distributed
              * according to a multivariate normal distribution.
              *
-             * @note For every dimension, this template and the corresponding implementation
-             *       have to be instantiated explicitly.
+             * @param cache         The Observable cache from which we draw the predictions.
+             * @param observables   The Observables whose distribution we model.
+             * @param mean          The vector of means.
+             * @param covariance    The covariance matrix
+             */
+            static LogLikelihoodBlockPtr MultivariateGaussian(ObservableCache cache, const std::vector<ObservablePtr> & observables,
+                    gsl_vector * mean, gsl_matrix * covariance, const unsigned & number_of_observations);
```

Comment:
I don't like this style `gsl_vector * mean`, it looks like a product to me. I'd suggest `gsl_vector* mean` or `gsl_vector *mean`. But that's general thing in the code

Reply:
I prefer it actually. Keeping it :-)